applications/nrf_desktop/doc/ble_qos.rst

```diff
@@ -50,7 +51,7 @@ Configuration channel options
 *****************************
 
 You can use the :ref:`nrf_desktop_config_channel` to configure the module or read the configuration.
-The module provides the following configuration options:
+The module is a configuration channel listener and provides the following configuration options:
```

Comment:
The module is called a `configuration channel listener` in case it is configurable through the configuration channel. This name is used in configuration channel dpocumentation. Can we leave it as it is ?